### PR TITLE
Mw feature navigation

### PR DESCRIPTION
**Pull request by MattWaraich**
# Navigation 

- Create routes/stacks in navigation
- Create nav bar 
- Add images for nav bar 
- Add active/inactive images for nav bar 
- Minor styling tint for active/inactive state 
- Create navigation screens
- Fixed bug in assets
- Add loader component

### CODE REVIEW

File: client/js/navigation/RootStackNavigator.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import {createAppContainer} from 'react-navigation';
+import NavigationLayout from './NavigationLayout';
+import {createStackNavigator} from 'react-navigation-stack';
+import AboutModal from '../screens/About';
+import ContactModal from '../screens/Contact';
+import FaqModal from '../screens/Faq';
+// import SearchModal from '../screens/Search';
```
Comment (siddharthparmar7) [inline client/js/navigation/RootStackNavigator.js]: Please remove the commented code.

File: client/js/navigation/RootStackNavigator.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export default createAppContainer(
+  createStackNavigator(
+    {
+      Layout: NavigationLayout,
+      //   Search: SearchModal,
```
Comment (siddharthparmar7) [inline client/js/navigation/RootStackNavigator.js]: same as above.

File: client/js/screens/About/About.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+const About = () => {
```
Comment (siddharthparmar7) [inline client/js/screens/About/About.js]: For best practice, please accept `props` as an argument for your presentation components.

File: client/js/screens/Booking/Booking.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+const Booking = () => {
```
Comment (siddharthparmar7) [inline client/js/screens/Booking/Booking.js]: same as above.

File: client/js/screens/Contact/Contact.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+const Contact = () => {
```
Comment (siddharthparmar7) [inline client/js/screens/Contact/Contact.js]: same as above.

File: client/js/screens/Faq/faq.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (siddharthparmar7) [inline client/js/screens/Faq/faq.js]: Make sure your file name is FAQ not faq. Please make the changes.

File: client/js/screens/Home/Home.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+const Home = () => {
```
Comment (siddharthparmar7) [inline client/js/screens/Home/Home.js]: add props as arg for your presentation component.

File: client/js/screens/Profile/Profile.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+const Profile = () => {
```
Comment (siddharthparmar7) [inline client/js/screens/Profile/Profile.js]: same as above.

File: client/js/screens/Saved/Saved.js [hunk 9]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+const Faves = () => {
```
Comment (siddharthparmar7) [inline client/js/screens/Saved/Saved.js]: same as above.

File: client/js/screens/Search/Search.js [hunk 10]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+const Search = () => {
```
Comment (siddharthparmar7) [inline client/js/screens/Search/Search.js]: same as above.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the changes.

File: client/js/navigation/RootStackNavigator.js (hunk 1 above)
Comment (MattWaraich) [reply client/js/navigation/RootStackNavigator.js]: completed.

File: client/js/navigation/RootStackNavigator.js (hunk 2 above)
Comment (MattWaraich) [reply client/js/navigation/RootStackNavigator.js]: completed.

File: client/js/screens/About/About.js (hunk 3 above)
Comment (MattWaraich) [reply client/js/screens/About/About.js]: completed.

File: client/js/screens/Faq/faq.js (hunk 6 above)
Comment (MattWaraich) [reply client/js/screens/Faq/faq.js]: completed.

File: client/js/screens/Home/Home.js (hunk 7 above)
Comment (MattWaraich) [reply client/js/screens/Home/Home.js]: completed.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**